app/src/data/ApiWrapper.ts

```diff
@@ -58,6 +78,10 @@ export class Fetcher<T> {
             JSON.stringify(item)
         )) as string
     }
+    
```

Comment:
This is correctly implemented, well done. Let's discuss whether this ultimately belongs here at this level of abstraction: Fetcher is currently a generic wrapper for a RESTful interface supporting Get, GetById, Put, Delete, Post. Adding a getByEmail() method could lead its users to trying that method on resources for which it does not apply, e.g. HostQuestion.